### PR TITLE
Use real CRS

### PR DESCRIPTION
**Pull request by thomas-waite**
## Summary
This PR configures the repo to use the real CRS that was generated in the Ignition ceremony. 

The `localSetupDatabase` has been populated with a subset of points for local testing. The URL from which points are fetched is still the usual `CloudFront` - this itself has been updated to point to the AWS S3 bucket containing the points calculated in the real setup.

### CODE REVIEW

File: packages/aztec.js/src/setup/index.js [hunk 1]
```diff
@@ -1,11 +1,4 @@
 /**
- * Read in points from the trusted setup points database.
- * NOTICE: THE TRUSTED SETUP IN THIS REPOSITORY WAS CREATED BY AZTEC ON A SINGLE DEVICE AND
- *   IS FOR TESTING AND DEVELOPMENT PURPOSES ONLY.
- *   We will be launching our multiparty computation trusted setup protocol shortly, where multiple entities
- *   create the trusted setup database and only one of them must act honestly in order for the setup database to be secure.
- *   If you wish to participate please let us know at hello@aztecprotocol.com
- *
```
Comment (ArnSch) [inline packages/aztec.js/src/setup/index.js:8]: 🎉

File: packages/aztec.js/src/setup/index.js [hunk 2]
```diff
@@ -14,10 +7,8 @@ const { constants } = require('@aztec/dev-utils');
 const BN = require('bn.js');
 const fetch = require('cross-fetch');
 
-const POINTS_DB_URL = 'https://ds8m7zxw3jpbz.cloudfront.net/data';
-
 const setup = {
-    POINTS_DB_URL: 'https://ds8m7zxw3jpbz.cloudfront.net/data',
+    POINTS_DB_URL: 'https://aztec-ignition.s3-eu-west-2.amazonaws.com/MAIN%20IGNITION/range_proofs/',
```
Comment (ArnSch) [inline packages/aztec.js/src/setup/index.js]: probably still worth putting behind cloudfront

**Review by ArnSch — APPROVED** [review]
Looks good ;)

File: packages/aztec.js/src/setup/index.js (hunk 2 above)
Comment (thomas-waite) [reply packages/aztec.js/src/setup/index.js]: done

**Comment by ArnSch** [comment]
Merged into `feat-optimisations`

**Pull request closed without merging**